common/core/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -221,10 +221,10 @@ namespace com.keyman.keyboards {
       return Lkc;
     }
 
-    public getSubkey(id: string): ActiveKey {
+    public getSubkey(coreID: string): ActiveKey {
       if(this.sk) {
         for(let key of this.sk) {
-          if(key.id == id) {
+          if(key.coreID == coreID) {
```

Comment:
This seems like a change in behaviour here? I note that this is called from `getKey` -- is this going to continue to work?

Reply:
Please refer to #5350.  This implements the same fix, just within the core's subkey-helper now.

Also refer to https://github.com/keymanapp/keyman/blob/7cbb78ea45dd2abdc3240de17a5b8eeed085f96d/web/source/kmwembedded.ts#L411-L418

(This block is removed as part of this PR, as it was basically a WET reproduction of the subkey-helper.)